tools/@aws-cdk/cdk-build-tools/bin/cdk-package.ts

```diff
@@ -65,7 +72,8 @@ async function main() {
   }
 
   if (options.post) {
-    await shell(options.post, { timers });
+    const commands = options.post.join(' && ');
+    await shell([commands], { timers });
```

Comment:
I've added a `pre` option (see above) and aligned the behavior with `cdk-build` to take a list of commands.